### PR TITLE
#227 - update rpm and add repo settings

### PR DESCRIPTION
**Pull request by olenagerasimova**
For #227 I've updated rpm-adapter, example repository yaml in readme and added repository storage configuration to `RpmSlice`.

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#242](https://github.com/artipie/artipie/issues/242) is assigned to @vatavuk[/z](https://www.0crat.com/u/vatavuk), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/c3caaf3a-a61e-4d01-ae6f-f6ac3c44a0b2); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: README.md [hunk 1]
```diff
@@ -197,6 +197,11 @@ repo:
   storage:
     type: fs
     path: /var/artipie/centos
+  settings:
+    digest: sha256 # Digest algorithm for rpm packages checksum calculation, sha256 (default) and sha1 are supported
+    naming-policy: sha1 # Naming policy for metadata files: plain (default), sha1 or sha256 prefixed
```
Comment (g4s8) [inline README.md:202]: @olenagerasimova can we use `sha256` as default (same as standard `createrepo` tool for RPM)?

**Comment by olenagerasimova** [comment]
@g4s8 thanks, sure, I've submitted https://github.com/artipie/rpm-adapter/pull/304 and corrected readme here

**Comment by Vatavuk** [comment]
@g4s8 please remove this issue from the scope

**Comment by olenagerasimova** [comment]
@0crat out

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @vatavuk[/z](https://www.0crat.com/u/vatavuk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request merged**